modules/cnf-cpu-infra-container.adoc

```diff
@@ -5,17 +5,55 @@
 [id="cnf-cpu-infra-container_{context}"]
 = Restricting CPUs for infra and application containers
 
-You can reserve cores (threads) from a single NUMA node for operating system housekeeping tasks and put your workloads on another NUMA node. Partitioning the CPUs this way can prevent the housekeeping processes from impacting latency-sensitive application processes. By default, CRI-O uses all online CPUs to run infra containers on nodes, which can result in context switches and spikes in latency. 
+Housekeeping and workloads use CPUs in a way that impacts latency-sensitive processes running on those CPUs. Partitioning the CPUs prevents the processes from interfering with each other. By default, CRI-O uses all online CPUs to run infra containers on nodes, which can result in context switches and spikes in latency. The following table describes how processes run on a CPU after you have tuned the node using the Performance Addon Operator:
 
-You can ensure that housekeeping tasks and workloads run on separate NUMA nodes by specifying two groups of CPUs in the `spec` section of the performance profile. 
+.Process' CPU assignments
+[%header,cols=2*] 
+|===
+|Process type
+|Details
 
-* `isolated` - The CPUs for the application container workloads. These CPUs have the lowest latency. Processes in this group have no interruptions and can, for example, reach much higher DPDK zero packet loss bandwidth.
+|Burstable and best-effort pods
+|Runs on any CPU except where low latency workload is running
 
-* `reserved` - The CPUs for the cluster and operating system housekeeping duties, including pod infra containers. Threads in the `reserved` group tend to be very busy, so latency-sensitive applications should be run in the `isolated` group. See link:https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed[Create a pod that gets assigned a QoS class of `Guaranteed`].
+|Infrastructure pods
+|Runs on any CPU except where low latency workload is running
+
+|Interrupts
+|Redirects to reserved CPUs (optional in {product-title} {product-version} and later)
+
+|Kernel processes
+|Pins to reserved CPUs
+
+|Latency sensitive workload pods
```

Comment:
Other renditions of this are latency-sensitive

Reply:
Updated as suggested.